docs/building.md

```diff
@@ -233,6 +233,105 @@ The `admin_address`, `guardian_address`, `governance_token_address`,
 `start_level` and `governance_token_id` are required values. The rest of the
```

Comment:
I just noticed that there is a mistake in the `make` call example for `treasuryDAO`: the `freeze_history` option is specified twice, please fix that as well.

Reply:
Fixed.